MekHQ/src/mekhq/campaign/mission/AtBScenario.java

```diff
@@ -1660,7 +1660,7 @@ public static String writeEntityWithCrewToXmlString(Entity tgtEnt, int indentLvl
             crew.append(MekHqXmlUtil.indentStr(indentLvl + 2)).append("<crewMember slot=\"")
                     .append(pos).append("\" name=\"").append(MekHqXmlUtil.escape(tgtEnt.getCrew().getName(pos)))
                     .append("\" nick=\"").append(MekHqXmlUtil.escape(tgtEnt.getCrew().getNickname(pos)))
-                    .append("\" gender=\"").append(tgtEnt.getCrew().getGender(pos))
+                    .append("\" gender=\"").append(tgtEnt.getCrew().getGender(pos).name())
```

Comment:
Is this the code that writes out a MUL entry? Can the MUL Parser handle this change?

Reply:
It can, using the same Gender::parseFromString method as Person does to load.